### ISSUE TITLE
Cannot rename variable from use variable name

### ISSUE THREAD

**Issue by dantleech** [phpactor/phpactor#829]
When renaming from a `use variable` as follows:

```        
return \Amp\call(function () use ($checkoutTask, $wo<>rkspace, $environment) {
```

we get the following error:

```
Error from Phpactor: Expected Variable or Parameter node, got "Microsoft\PhpParser\Node\UseVariableName
```

Note that I think currently variable renaming is limited to the current scope (probably the function) but in this case we also need to include the parent scope.

**Comment by elythyr** [comment]
What is a "use variable" ?

**Comment by dantleech** [comment]
Updated the description :)

**Comment by elythyr** [comment]
I hate specific :D

**Comment by dantleech** [comment]
Fixed in #888
